### DISCUSSION TITLE
Clean and safe phone to begin

### DISCUSSION THREAD

**Discussion by TrentJason** [Divested-Mobile/DivestOS-Build#343] [Q&A]
I bought a used Xperia XZ2 compact with Lineage OS already installed. I was about to replace it with DivertOS but then I got suspicious. What if the previous owner installed somekind of backdoor somewhere deep in to this phones memory and now is just waiting me to expose my personal data.
I'm asking that will the install process of DivertOS wipe the phone so completely that there's nothing left to worry about? Or is there some procedure I should do to ensure clean and safe phone to begin.

**Comment by SkewedZeppelin** [comment]
you can flash stock first and then your rom of choice, but even then it still isn't a guarantee

**Comment by TrentJason** [comment]
Does that mean that if I buy any used phone even with stock Android I have no way to know if there's some backdoor installed? There isn't any indicators that reveals tampering?

**Reply by SkewedZeppelin** [reply]
you can verify stock is correct on some devices, see the GrapheneOS Attestation app: https://attestation.app/about

**Reply by TrentJason** [reply]
I'll check that. Thank you for all the help.
